scripts/update_specs_from_pricing.py

```diff
@@ -191,18 +191,71 @@ def get_rds_pricing():
         FormatVersion='aws_v1',
     )
 
+    product_map = {
```

Comment:
Can be organise the list a bit clearer? Really sort on "ID" or on the database name? Just for transparancy

Reply:
Done